From 3a87d65245819b25fe397f959ea639f5b1bef59a Mon Sep 17 00:00:00 2001
From: Bharat Kumar Gogada <bharat.kumar.gogada@xilinx.com>
Date: Mon, 13 Jan 2020 15:26:37 +0530
Subject: [PATCH 0841/1851] pci: xdma: pl:: Remove unused variable

commit 03c385dd66fa4a9997c68c5495e5a3c8505d2396 from
https://github.com/Xilinx/linux-xlnx.git

Removing unsued variable in tst_bit in xilinx_irq_domain_alloc.

Signed-off-by: Bharat Kumar Gogada <bharat.kumar.gogada@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: not-upstreamable
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/pci/controller/pcie-xdma-pl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/controller/pcie-xdma-pl.c b/drivers/pci/controller/pcie-xdma-pl.c
index 1005df802fd3..e3a09d130fe2 100644
--- a/drivers/pci/controller/pcie-xdma-pl.c
+++ b/drivers/pci/controller/pcie-xdma-pl.c
@@ -476,7 +476,7 @@ static int xilinx_irq_domain_alloc(struct irq_domain *domain, unsigned int virq,
 {
 	struct xilinx_pcie_port *pcie = domain->host_data;
 	struct xilinx_msi *msi = &pcie->msi;
-	int bit, tst_bit;
+	int bit;
 	int i;
 
 	mutex_lock(&msi->lock);
-- 
2.31.1

